release/windows-installer/src/WindowsInstaller.hs

```diff
@@ -13,18 +13,30 @@ import SdkVersion
 
 main :: IO ()
 main = do
-    [installerFile, sdkDir] <- getArgs
-    writeFile installerFile $ nsis $ installer sdkDir
+    [installerFile, sdkDir, logo] <- getArgs
+    writeFile installerFile $ nsis $ installer sdkDir logo
 
-installer :: FilePath -> Action SectionId
-installer sdkDir = do
-    name "DAML SDK installer"
+installer :: FilePath -> FilePath -> Action SectionId
+installer sdkDir logo = do
+    name "DAML SDK"
     outFile "daml-sdk-installer.exe"
+    installIcon (fromString logo)
     requestExecutionLevel User
+    unsafeInjectGlobal "!insertmacro MUI_PAGE_WELCOME"
```

Comment:
I'm just adding this to nsis-0.3.3 (no need to upgrade to it though - can be a later date)